test/core/end2end/gen_build_json.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.7
```

Comment:
For what it's worth, I'm told that the weird python=python3 with python2 installed configuration is against recommendation, so dropping the explicit selection of python2 here as you are doing is probably fine.